edb/server/protocol/auth_ext/http_client.py

```diff
@@ -50,7 +50,7 @@ async def post(  # type: ignore[override]
         **kwargs: Any,
     ) -> httpx.Response:
         if self.edgedb_orig_base_url:
-            path = f'{self.edgedb_orig_base_url}/{path}'
+            path = f'{self.edgedb_orig_base_url}{path}'
```

Comment:
Our HTTP test URLs had an extra `/` in them due to how we were parsing these URLs. Not really a bug, but when I went to write new tests, I realized that these URLs were wrong, so just fixed this while I was in here. Resulting in some additional churn here.